stl.go

```diff
@@ -215,15 +228,35 @@ func ReadFromSTL(i io.Reader) (o *Subtitles, err error) {
 			continue
 		}
 
+		justification := parseJustificationAttribute(t.justificationCode)
+		rows := bytes.Split(t.text, []byte{stlLineSeparator})
+
+		position := STLPosition{
+			MaxRows:          g.maximumNumberOfDisplayableRows,
+			Rows:             len(rows),
+			VerticalPosition: t.verticalPosition,
+		}
+
+		styleAttributes := StyleAttributes{
+			STLJustification: &justification,
+			STLPosition:      &position,
+		}
+		styleAttributes.propagateSTLAttributes()
+
 		// Create item
 		var i = &Item{
-			EndAt:   t.timecodeOut - g.timecodeStartOfProgramme,
-			StartAt: t.timecodeIn - g.timecodeStartOfProgramme,
+			EndAt:       t.timecodeOut - g.timecodeStartOfProgramme,
+			InlineStyle: &styleAttributes,
+			StartAt:     t.timecodeIn - g.timecodeStartOfProgramme,
 		}
 
 		// Loop through rows
-		for _, text := range bytes.Split(t.text, []byte{0x8a}) {
-			parseTeletextRow(i, ch, func() styler { return newSTLStyler() }, text)
+		for _, text := range bytes.Split(t.text, []byte{stlLineSeparator}) {
+			if g.displayStandardCode == stlDisplayStandardCodeOpenSubtitling {
+				parseOpenSubtitleRow(i, ch, func() styler { return newSTLStyler() }, text)
```

Comment:
You should process the error returned by this function by at least logging it or this method should not return an error

Reply:
Done.